.github/workflows/iceberg_extension_distribution.yaml

```diff
@@ -0,0 +1,445 @@
+# Reusable workflow for building DuckDB extensions using a standardized environment
+#
+# The workflow:
+#   - builds the extension using the CI workflow from the corresponding DuckDB version
+#   - uploads the extensions as gh actions artifacts in the following format:
+#         <ext_name>-<duckdb_version>-extension-<arch><optional_postfix>
+#
+# note: extensions are simply uploaded to GitHub actions, deploying the extensions is done a separate step. More info on
+#       this can be found in https://github.com/duckdb/extension-template
+
+name: Iceberg Extension Distribution
+on:
+  workflow_call:
```

Comment:
do we really need all these inputs?can we set these as variables or so? I am not sure we will need to run this workflow with any of htese inputs